### PR TITLE
Added random alias generator

### PR DESCRIPTION
**Pull request by ShootGan**
As in #3  i added random string generator for alias.

### CODE REVIEW

**Comment by aditeyabaral** [comment]
Wouldn't you want to make the PR after Hacktoberfest starts? (I am not aware if it has started in any country)

**Comment by ShootGan** [comment]
> Wouldn't you want to make the PR after Hacktoberfest starts? (I am not aware if it has started in any country)

Yes, that's a good idea. But I'm not sure how Hacktoberfest works. I have to create a new pull request in October, right?

**Comment by aditeyabaral** [comment]
Yup. Pull requests count only from October 1st :)

**Comment by ShootGan** [comment]
ok, so I'm closing this pull request now and I will do a new in October. Thanks for help  :D

File: app/redirector.py [hunk 1]
```diff
@@ -38,6 +40,14 @@ def checkAliasExists(alias_url):
     return False
 
 
+def generate_alias() -> str:
```
Comment (aditeyabaral) [inline app/redirector.py]: Change the name to `generateAlias()`

**Comment by aditeyabaral** [comment]
@ShootGan I have added a comment. Update the code and I will merge.

**Comment by aditeyabaral** [comment]
Looks good!

**Pull request merged**